includes/definitions/stoneos.yaml

```diff
@@ -0,0 +1,19 @@
+os: stoneos
+text: Hillstone StoneOs
+type: network
+icon: hillstone
+group: hillstone
+over:
+    -
+        graph: device_bits
+        text: Device Traffic
+    -
+        graph: device_processor
+        text: CPU Usage
+    -
+        graph: device_mempool
+        text: Memory Usage
+discovery:
+    -
+        sysObjectId: .1.3.6.1.4.1.28557.1
```

Comment:
Do you need sysDescr as well? Will sysObjectID not be unique enough?

Reply:
I think only Hillstone use this Oid, removed sysdescr match.